Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -987,6 +988,8 @@ def define_Configs(self):
         self.REPACKDefaultSeq='DigiToRawRepack'
         self.PATDefaultSeq='miniAOD'
         self.PATGENDefaultSeq='miniGEN'
+        #TODO: Check based of file input
```

Comment:
Hi Kenneth - did you have an idea how to do this? I've been running the GEN and NANOGEN steps independently due to different CMSSW versions, and so had to change this logic for my use case. I see `cmsDriverOptions.py` has a `--inputEventContent` argument, but I don't know if requiring that argument is ideal.

Reply:
no, I don't think that's possible. But for GEN it really doesn't matter, I just use the 10_6 or 11_X for everything.

(assuming you're using a gridpack, it doesn't matter at all, and you probably don't care tooooo much about the Pythia version)

Reply:
By the way you can see a lot of configs and cmsDriver commands that I've used here:

https://github.com/kdlong/WMassNanoGen

Reply:
Thanks, the scripts are helpful!